### ISSUE TITLE
outdated MET code to be removed

### ISSUE THREAD

**Issue by jpata** [cms-sw/cmssw#37975]
The following MET modules cannot run because they use functionality removed from CMSSW. 
```
RecoMET/METPUSubtraction:
  NoPileUpPFMEtDataProducer:
  - 'NoPileUpPFMEtDataProducer::produce(edm::Event &, const edm::EventSetup &); JetCorrector::getJetCorrector(const std::basic_string<char, std::char_traits<char>, std::allocator<char> > &, const edm::EventSetup &); '
```

It looks like `NoPileUpPFMEtDataProducer` is outdated and should be removed (along with the python configs).

@cms-sw/jetmet-pog-l2 @michaelwassmer

**Comment by jpata** [comment]
assign reconstruction

**Comment by jpata** [comment]
type jetmet

**Comment by mandrenguyen** [comment]
@michaelwassmer 
Any thoughts on this issue?

**Comment by michaelwassmer** [comment]
Sorry, I don't really know what this module is doing. This is way from before my time. However, if the functionality doesn't exist anymore, there's probably no reason to keep it. Furthermore, in the header file of the module, the one-line summary says something about MVA MET, which as far as I know is not used anywhere.

**Comment by alkaloge** [comment]
+1

**Comment by rappoccio** [comment]
This is addressed by #40175, closing.
